hardhat.config.ts

```diff
@@ -8,17 +8,17 @@ import { askDeployer, askFees, hardhatDefaultConfig, deploy } from "@defi.org/we
 import _ from "lodash";
 import "hardhat-watcher";
 import "solidity-coverage";
-import BigNumber from "bignumber.js";
 
-import { deployUninsuredVestingV1 } from "./deployment/uninsured-vesting-v1";
+import { deployUninsuredVestingV1, ConfigTuple as UninsuredConfig } from "./deployment/uninsured-vesting-v1";
+import { deployInsuredVestingV1, ConfigTuple as InsuredConfig } from "./deployment/insured-vesting-v1";
 
 type Contract = "Insured" | "Uninsured";
 
 task("deploy-contract", "Deploy Excited contract")
```

Comment:
I personally would prefer a separate task for each contract, to be more explicit

Reply:
Do you really think that's necessary? This way, code is pretty DRY, and in order to deploy you have to pass an explicit contract name AND the dry false flag, eg:
`npx hardhat deploy-contract --contract Uninsured --dry false`